### DISCUSSION TITLE
Project Status?

### DISCUSSION THREAD

**Discussion by holzmaster** [test-summary/action#50] [Q&A]
I'm considering to contribute and it seems that the project has stalled. Are contributions still accepted?

**Comment by ethomson** [comment]
Hi @holzmaster - sorry about the delay. Absolutely, we'd accept a contribution!

**Comment by gstokkink** [comment]
@ethomson hi, thanks for creating this project 😄 I've opened three PRs, could you please take a look when you have the time? Cheers!

**Comment by andreyfel** [comment]
@ethomson this is a great action! Thank you for it! It would be nice to get the Node16 deprecation resolved. Do you have a chance to look at @gstokkink's PRs?
